tests/components/twitch/test_sensor.py

```diff
@@ -80,6 +80,7 @@ async def test_oauth_with_sub(
     sensor_state = hass.states.get(ENTITY_ID)
     assert sensor_state.attributes["subscribed"] is True
     assert sensor_state.attributes["subscription_is_gifted"] is False
+    assert sensor_state.attributes["subscription_tier"] == 1000
```

Comment:
But, shouldn't we show 1/2/3 here?

Reply:
I don't think we should change the value at all.  The end user would want to get it as the API provides it, I would think.  We only show the 1000, 2000, and 3000 as-is.

Reply:
Yes, but users don't talk to an API, users go on Twitch and have a Tier 1 sub, so they expect their tier to be `1`. If I don't have knowledge of the API, seeing myself being a `tier 1000` sub just feels off

Reply:
Fair enough.  I'll look at it.